pkg/cmd/codespace/code.go

```diff
@@ -31,18 +31,12 @@ func newCodeCmd(app *App) *cobra.Command {
 
 // VSCode opens a codespace in the local VS VSCode application.
 func (a *App) VSCode(ctx context.Context, codespaceName string, useInsiders bool) error {
-	if codespaceName == "" {
-		codespace, err := chooseCodespace(ctx, a.apiClient)
-		if err != nil {
-			if err == errNoCodespaces {
-				return err
-			}
-			return fmt.Errorf("error choosing codespace: %w", err)
-		}
-		codespaceName = codespace.Name
+	codespace, err := getOrChooseCodespace(ctx, a.apiClient, codespaceName)
+	if err != nil {
+		return fmt.Errorf("get or choose codespace: %w", err)
```

Comment:
This isn't a particularly user-friendly error message. "get or choose codespace" is really just a function name, rendered with spaces: it's an internal detail of the structure of the program. The error message from getOrChooseCodespaces should be good enough to explain itself, so that you can simply return err here (as you do later on in this PR).

Reply:
I agree. I just copied this from the other places where we call `getOrChooseCodespace`. I'll go through and change all of them.